server/views/components/actions-card/test.njk

```diff
@@ -0,0 +1,6 @@
+<html>
+  <body>
+      {% from "components/actions-card/macro.njk" import actionsCard %}
+      {{ actionsCard(actions) }}
```

Comment:
```
actionsCard(actions)
```
By passing in `actions` as an argument, does this mean the controller (or something?) is responsible for putting an array of actions in the data for the `res.render` call? 🤔 

I can see that working for things that the controller/routes are responsible for providing to the page (eg: prisoner data etc); but does this feel right for a list of actions that won't change? 🤔

Reply:
You can also inline when it's hardcoded like our list of actions will be. The reason it's like this is so the model is exercised, and thus guaranteed to be type safe.

Reply:
Oh yeah, good point 👍